### PR TITLE
feat(404): init page

### PR DESCRIPTION
**Pull request by m0ksem**
closes https://github.com/epicmaxco/vuestic-admin/issues/961

image
image

### CODE REVIEW

**Comment by m0ksem** [comment]
@asvae, are we going to remove old 404 pages, or we store them just in case?

**Comment by asvae** [comment]
@m0ksem yes, please remove

File: src/router/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import UIRoute from '../pages/admin/ui/route'
 const routes: Array<RouteRecordRaw> = [
   {
     path: '/:pathMatch(.*)*',
-    redirect: { name: 'dashboard' },
+    component: () => import('../pages/404.vue'),
```
Comment (asvae) [inline src/router/index.ts:13]:
```suggestion
    redirect: { name: 'dashboard' },
```

File: src/router/index.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import UIRoute from '../pages/admin/ui/route'
 const routes: Array<RouteRecordRaw> = [
   {
     path: '/:pathMatch(.*)*',
-    redirect: { name: 'dashboard' },
```
[the remark this answers is not in the document]
Comment (asvae) [reply src/router/index.ts]: Discussed with vasily to keep old approach

**Pull request merged**